Week1/homework/app.js

```diff
@@ -1,11 +1,144 @@
+    {
+      id: 'the_gulen_movement',
+      title: 'The Gulen Movement',
+      language: 'English',
+      author: 'Muhammet Cetin',
+    },
+    {
+      id: 'jip_en_janneke',
+      title: 'Jip en Janneke',
+      language: 'Dutch',
+      author: 'Annie M. G. Schmidt',
+    },
+    { id: 'bahar_nesidesi', title: 'Bahar Nesidesi', language: 'Turkish', author: 'M. F. Gulen' },
+    { id: 'sozler', title: 'Sozler', language: 'Turkish', author: 'B. Said Nursi' },
+  ];
+
+  const imagesOfBook = [
+    { towards_to_lost_paradise: './img/towards_to_lost_paradise.jpg' },
+    { light: './img/light.jpg' },
+    { bizim_firavun: './img/bizim_firavun.jpg' },
+    { hayatin_icinden: './img/hayatin_icinden.jpg' },
+    { kalbimin_efendisi: './img/kalbimin_efendisi.jpg' },
+    { mesel_denizi: './img/mesel_denizi.jpg' },
+    { the_gulen_movement: './img/the_gulen_movement.jpg' },
+    { jip_en_janneke: './img/jip_en_janneke.jpg' },
+    { bahar_nesidesi: './img/bahar_nesidesi.jpg' },
+    { sozler: './img/sozler.jpg' },
+  ];
+
+  const altImage = [
+    { towards_to_lost_paradise: 'book cover for towards_to_lost_paradise' },
+    { light: 'book cover for light' },
+    { bizim_firavun: 'book cover for bizim_firavun' },
+    { hayatin_icinden: 'book cover for hayatin_icinden' },
+    { kalbimin_efendisi: 'book cover for kalbimin_efendisi' },
+    { mesel_denizi: 'book cover for mesel_denizi' },
+    { the_gulen_movement: 'book cover for the_gulen_movement' },
+    { jip_en_janneke: 'book cover for jip_en_janneke' },
+    { bahar_nesidesi: 'book cover for bahar_nesidesi' },
+    { sozler: 'sozler' },
+  ];
+  console.log(myLast10Books);
+  console.log(
+    idsOfBook +
+      'This is for debugging and making sure everything is in order. Delete it later when you are done :',
+  );
+  console.log(detailsOfBook);
+
+  // 1.3
+  // function render(ul) {
+  //   for (let i = 0; i < titlesOfBook.length; i++) {
+  //     const listOfIds = idsOfBook[i];
+  //     const li = document.createElement('li');
+  //     ul.appendChild(li);
+  //     li.textContent = listOfIds;
+  //     li.className = 'listOfIds';
+  //   }
+  // }
+
+  // 1.5
+  function render(ul) {
+    for (let i = 0; i < detailsOfBook.length; i++) {
+      const listOfBooks = detailsOfBook[i].title;
+      const listOfAuthors = detailsOfBook[i].author;
+      const listOfLanguage = detailsOfBook[i].language;
+      const bookImage = Object.values(imagesOfBook[i]);
+      const listAltImage = Object.values(altImage[i]);
+
+      const li = document.createElement('li');
+      const h2 = document.createElement('h2');
+      const p = document.createElement('p');
+      const img = document.createElement('img');
+
+      li.textContent = listOfBooks;
+      h2.textContent = listOfAuthors;
+      p.textContent = listOfLanguage;
+      img.src = bookImage;
+      img.alt = listAltImage;
+
+      ul.appendChild(li);
+      li.appendChild(h2);
+      h2.appendChild(p);
```

Comment:
With `appendChild` you put the element inside the element you specify at the beginning, so in this case you put the `p` tag inside the `h2` title. Right now you have the `p` tag and the image `img` inside the title. Better would be to create a container `div` and put all the elements inside that container of in this case you could append everything to the `li` element since it is already the container for the elements.

Reply:
Hi @paulbremer 
I noticed that one after finishing the homework then I decide to search and find a way to resolve the issue but that week there were some additional task which I have to work on it then I left it like that.  I will work on it very soon.
Thanks million times for your great efforts.